lib/screens/dashboard.dart

```diff
@@ -12,40 +12,114 @@ class Dashboard extends GetView<RobotStateController> {
   Dashboard({super.key});
 
   final RemoteController remoteControl = Get.find();
+  
+  // Helper method to format the robot status text
+  String _getStatusText(String status) {
```

Comment:
Maybe it would be more accurate to change it like this:
```suggestion
  String _formatStatusText(String status) {
```

it does not `get`, it `formats` provided String.